<ui version="4.0" >
 <class>mdlgSelectVars</class>
 <widget class="QDialog" name="mdlgSelectVars" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>420</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>11</x>
     <y>10</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Select Variables :</string>
   </property>
  </widget>
  <widget class="Line" name="line" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>481</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Available Columns:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="mListViewCols" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>251</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mBtnZ" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>230</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>>></string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mLineEditY" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>180</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>260</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Charge State : (Optional)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mLineEditX" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>130</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="mLineEditPepSeq" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>80</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mBtnX" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>130</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>>></string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>60</y>
     <width>147</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Peptide Sequence : (Req.)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mLineEditCS" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>280</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mBtnY" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>180</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>>></string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mLineEditZ" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>230</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mBtnPepSeq" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>80</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>>></string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>110</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>X variable : (Req.)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>160</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Y Variable : (Optional)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mBtnCS" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>280</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>>></string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>210</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Z Variable : (Optional)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>310</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Tryptic State : (Optional)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mLineEditTrypState" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>330</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mBtnTrypState" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>330</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>>></string>
   </property>
  </widget>
  <widget class="QToolBox" name="mtoolBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>370</y>
     <width>251</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="CSstates" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>249</width>
      <height>113</height>
     </rect>
    </property>
    <attribute name="label" >
     <string>Select Charge States:</string>
    </attribute>
    <widget class="QCheckBox" name="mchkBoxCSother" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>90</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>Charge State 4 and above</string>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="mchkBoxCS3" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>60</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>Charge State 3</string>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="mchkBoxCS2" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>30</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>Charge State 2</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="mchkBoxCS1" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>0</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>Charge State 1</string>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="TrState" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>249</width>
      <height>113</height>
     </rect>
    </property>
    <attribute name="label" >
     <string>Select Tryptic States:</string>
    </attribute>
    <widget class="QCheckBox" name="mchkBoxFull" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>80</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>Full</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="mchkBoxNone" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>0</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>None</string>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="mchkBoxPartial" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>40</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>Partial</string>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mdlgSelectVars</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
